spinalcordtoolbox/labels.py

```diff
@@ -267,6 +267,34 @@ def label_vertebrae(img: Image, vertebral_levels: Sequence[int] = None) -> Image
     return out
 
 
+def check_missing_label(img, ref):
+    """
+    Function that return the list of label that are present in ref and not in img.
+    This is useful to compute false negative (label that are in ref and not img) and false positive
+    (labels that are present in img and not ref)
```

Comment:
Nit: This part of the docstring might need to be updated too. (Re: comments about 2 lists, FP/FN)

Reply:
Done :)